tileview/src/main/java/com/qozix/tileview/widgets/ZoomPanLayout.java

```diff
@@ -379,15 +380,23 @@ public void smoothScaleTo( float destination ) {
    * @param focusX The horizontal focal point to maintain, relative to the screen (as supplied by MotionEvent.getX).
    * @param focusY The vertical focal point to maintain, relative to the screen (as supplied by MotionEvent.getY).
    * @param scale The final scale value the ZoomPanLayout should animate to.
+   * @param origination The optional origination of the scale change (can be null). Values from {@link com.qozix.tileview.widgets.ZoomPanLayout.ZoomPanListener.Origination}
    */
-  public void smoothScaleFromFocalPoint( int focusX, int focusY, float scale ) {
+  public void smoothScaleFromFocalPoint( int focusX, int focusY, float scale, @Nullable ZoomPanListener.Origination origination ) {
```

Comment:
you're totally correct, my mistake, i just saw in the diff that the original signature had a - next to it, and was replaced by your new signature, but when i look at the complete file i do see that they are new signatures, and the old signature pipes through, so it is _not_ a breaking change.  that said, not sure i love passing origination around, especially if it's null, seems like it might be a little misleading - but, i don't hate it either - more on overall approach later.